### ISSUE TITLE
Minification issue

### ISSUE THREAD

**Issue by tquetano-r7** [webpack-contrib/style-loader#191]
**Do you want to request a *feature* or report a *bug*?**

**What is the current behavior?**

When build in minification, it appears that in certain scenarios an additional `"` is added at the end of an `rgba` declaration. It is very hard to reproduce with a simple example, but confirmed that the built version in the file has a standard declaration (such as `rgba(0,0,0,0.5)`) and when injected into the DOM it adds a `"` at the end, e.g. `rgba(0,0,0,0.5")`. It is sparingly, so it is tough to pinpoint an exact cause, but I have confirmed with eyeball reconciliation that between the stored version in the file and the rendered version in the DOM this injection occurs. Sorry I can't offer more details, but reverting to 0.13.2 it operates correctly, so I wonder if the "relative URL" fix that was added somehow created this.

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as your webpack version, Node.js version and Operating System.**

webpack version = 2.2.1
node version = 6.9.4
os version = manjaro linux

**Comment by tquetano-r7** [comment]
so doing a bit more digging, it actually does look specific to URLs ... the example here is related to the `filter` fallback of inline SVG formation. Here it is in the file:

```css
filter:url(\'data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg"><filter id="filter"><feGaussianBlur in="SourceAlpha" stdDeviation="5" /><feOffset dx="1" dy="1" result="offsetblur" /><feFlood flood-color="rgba(0,0,0,0.5)" /><feComposite in2="offsetblur" operator="in" /><feMerge><feMergeNode /><feMergeNode in="SourceGraphic" /></feMerge></filter></svg>#filter\')
```
And in the DOM:

```css
filter:url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg"><filter id="filter"><feGaussianBlur in="SourceAlpha" stdDeviation="5" /><feOffset dx="1" dy="1" result="offsetblur" /><feFlood flood-color="rgba(0,0,0,0.5")" /><feComposite in2="offsetblur" operator="in" /><feMerge><feMergeNode /><feMergeNode in="SourceGraphic" /></feMerge></filter></svg>#filter')
```

Notice the `rgba` declaration in `flood-color`. Certainly explains why I wasn't seeing it very much, it appears to only happen when encased in a URL string.

**Comment by michael-ciniawsky** [comment]
Discussion moved to #192 Thx 😛

**Comment by d3viant0ne** [comment]
Leave issue open until the pull request lands, it can be closed from within the pull request by editing the initial users PR comment with `Closes #191`
